data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -1060,6 +1112,20 @@
     "description": "A massive rhino-like dinosaur with a tall bony crest from which four long horns and a short nose horn emerge.",
     "reproduction": { "baby_egg": "egg_pentaceratops", "baby_count": 20, "baby_timer": 24 }
   },
+  {
+    "type": "MONSTER",
+    "id": "mon_kosmoceratops",
+    "name": { "str_sp": "Kosmoceratops" },
+    "copy-from": "mon_triceratops",
+    "melee_skill": 9,
+    "armor_cut": 7,
+    "armor_bash": 8,
+    "hp": 670,
+    "zombify_into": "mon_zosmoceratops",
+    "description": "A cousin of the Triceratops, This beast has returned from extinction, and although it's a herbivore it can do alot of damage when angered.",
```

Comment:
```suggestion
    "description": "A cousin of the Triceratops, this beast has returned from extinction.  Although it's a herbivore, it looks like it can do a lot of damage if angered.",
```